src/Entity/CitizenProject.php

```diff
@@ -672,4 +672,18 @@ public function setNextAction(CitizenAction $citizenAction): void
     {
         $this->nextAction = $citizenAction;
     }
+
+    public function exportCommitteeSupports(): string
+    {
+        return join(', ', array_map(function (CitizenProjectCommitteeSupport $committeeSupport) {
+            return $committeeSupport->getCommittee()->getName();
+        }, $this->committeeSupports->toArray()));
+    }
+
+    public function exportSkills(): string
+    {
+        return join(', ', array_map(function (CitizenProjectSkill $skill) {
```

Comment:
`return join(', ', $this->skills->toArray());

thanks to `BaseSkill::__toString()`

Reply:
Nice catch ! :+1: Thx